src/Contao/View/Contao2BackendView/EditMask.php

```diff
@@ -642,16 +642,21 @@ protected function getHeadline()
      */
     protected function doPersist()
     {
-        $environment   = $this->getEnvironment();
-        $dataProvider  = $environment->getDataProvider($this->model->getProviderName());
-        $inputProvider = $environment->getInputProvider();
+        $environment     = $this->getEnvironment();
+        $dataProvider    = $environment->getDataProvider($this->model->getProviderName());
+        $inputProvider   = $environment->getInputProvider();
+        $editInformation = System::getContainer()->get('cca.dc-general.edit-information');
 
         if (!$this->model->getMeta(ModelInterface::IS_CHANGED)) {
             return true;
         }
 
         $this->handlePrePersist();
 
+        if ($editInformation->hasAnyModelError()) {
```

Comment:
```suggestion
        if ($editInformation->getModelError($this->model)) {
```
You are only interested in the errors within the current model and not all models being contained in the system, aren't you?

Reply:
you could be right ...

Reply:
fixed